app/services/album_service.rb

```diff
@@ -0,0 +1,15 @@
+module AlbumService
+  def self.upload_images(album, images, photographer, name)
```

Comment:
Unused method argument - `images`. If it's necessary, use `_` or `_images` as an argument name to indicate that it won't be used.<br>Unused method argument - `photographer`. If it's necessary, use `_` or `_photographer` as an argument name to indicate that it won't be used.<br>Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used.